azure_translator/__init__.py

```diff
@@ -57,19 +57,24 @@ def translate(self, text, to=DEFAULT_LANGUAGE, source_language=None):
         :param source_language:    optional source language
         """
         params = {
-            'text': text,
-            'to': to,
+            'to': to
         }
+
+        body = [{
+            'text' : text
+        }]
         if source_language:
             params['from'] = source_language
         try:
-            resp = requests.get(
+            resp = requests.post(
                 self.TRANSLATE_API,
                 headers={
                     'Authorization': 'Bearer {}'.format(self.get_access_token()),
```

Comment:
Authorization changes, put a header `Ocp-Apim-Subscription-Key` with the value `self.api_key`

Reply:
according to the doc, you don't need the Ocp-Apim-Subscription-Key in the second request, infact there is 2 ways to do it:
  - either provide the key and all the params in one request 
  - or, as we used to do, get the jwt authorization token through our {self.api_key} and then keep using it through the Authorization header, (I went with the second way since it's basically how our sdk functions already).

Reply:
in any case, I did test the current implementation for the version 3 translation api and it functions properly